src/main/java/org/opensearch/ad/transport/DeleteModelTransportAction.java

```diff
@@ -113,7 +114,15 @@ protected DeleteModelNodeResponse nodeOperation(DeleteModelNodeRequest request)
         String adID = request.getAdID();
         LOG.info("Delete model for {}", adID);
         // delete in-memory models and model checkpoint
-        modelManager.clear(adID);
+        modelManager
+            .clear(
+                adID,
+                ActionListener
+                    .wrap(
+                        r -> LOG.info("Deleted model for [{}] with response [{}] ", adID, r),
```

Comment:
When will we call this delete model action? Will it cause too many log lines for HC detector?

Reply:
We call it when people stop or delete a detector.  So it should not cause too many log lines, right?

Reply:
Yes, that's ok.